scripts/remove_alpha.py

```diff
@@ -0,0 +1,15 @@
+import fileinput
+import os
+
+
+version_file = os.getenv("VERSION_FILE")
+if not os.path.isfile(version_file):
+    raise ValueError(f"environment variable `VERSION_FILE` is not a file: {version_file}")
+
+alpha_var_name = "VERSION_ALPHA"
+
+for line in fileinput.input(version_file, inplace=True):
+    if line.startswith(alpha_var_name):
+        print(f"{alpha_var_name} = 0")
+    else:
+        print(line.rstrip('\n'))
```

Comment:
Is this used anywhere? Seems like a very specific use case

Reply:
Not seen it used once. But was a part of the template repo, so i converted it. 🤷